docs/types/reference-types.rst

```diff
@@ -165,31 +165,40 @@ always use one of the value types ``bytes1`` to ``bytes32`` because they are muc
     that you are accessing the low-level bytes of the UTF-8 representation,
     and not the individual characters.
 
-.. index:: ! bytes-concat
+.. index:: ! bytes-concat, ! string-concat
 
 .. _bytes-concat:
+.. _string-concat:
 
-``bytes.concat`` function
-^^^^^^^^^^^^^^^^^^^^^^^^^
+The functions ``bytes-concat`` and ``string-concat``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can concatenate a variable number of ``bytes`` or ``bytes1 ... bytes32`` using ``bytes.concat``.
-The function returns a single ``bytes memory`` array that contains the contents of the arguments without padding.
+You can concatenate a arbitrary number of ``string`` values using ``string.concat``.
+The function returns a single ``string memory`` array/array-slices that contains the contents of the arguments without padding.
```

Comment:
The arguments can be strings or string slices, not the return value.

Reply:
Maybe just add to the previous sencence: You can use anything that can be implicitly converted to `string memory`, which also includes string slices and string literals.